package/crds/github.crossplane.io_providerconfigs.yaml

```diff
@@ -49,6 +47,19 @@ spec:
           spec:
             description: A ProviderConfigSpec defines the desired state of a ProviderConfig.
             properties:
+              client:
+                description: ProviderClient defines GitHub base and upload URL
+                properties:
+                  baseUrl:
+                    description: Base URL of GitHub Enterprise
+                    type: string
+                  uploadUrl:
+                    description: Upload URL of GitHub Enterprise
+                    type: string
+                required:
+                - baseUrl
+                - uploadUrl
```

Comment:
I believe we're not really using uploadUrl - should we consider leaving it off the interface and supplying baseUrl for both parameters to NewEnterpriseClient?

Reply:
I am in dilemma of this. Reason is I am planning to raise a PR to apply this change to upstream (https://github.com/crossplane-contrib/provider-github). In that PR, I will probably use two fields instead of one.

I think if this can be fixed in upstream, there is no strong reason for us to keep CBA fork.

Reply:
Hi @davidgibb, I changed the code to make uploadURL optional. Here is the commit: https://github.com/12kmps/crossplane-provider-github/pull/12/commits/b18283ac3692f4505deff38833f4c776e907317d